crates/engine/primitives/src/lib.rs

```diff
@@ -49,13 +49,11 @@ pub use config::*;
 /// payload job. Hence this trait is also [`PayloadTypes`].
 pub trait EngineTypes:
     PayloadTypes<
-        BuiltPayload: TryInto<Self::ExecutionPayloadEnvelopeV1>
-                          + TryInto<Self::ExecutionPayloadEnvelopeV2>
-                          + TryInto<Self::ExecutionPayloadEnvelopeV3>
-                          + TryInto<Self::ExecutionPayloadEnvelopeV4>,
-    > + DeserializeOwned
-    + Serialize
-    + 'static
```

Comment:
cool, I believe those were necessary due to some jsonrpsee requirements at some point